helpers/index.ts

```diff
@@ -11,15 +11,7 @@ export {
   recoverPublicKey,
 } from "./crypto";
 export { decodeHexString } from "./decodeHexString";
-export {
-  commify,
-  formatBytes32String,
-  formatEther,
-  parseEther,
-  randomBytes,
-  solidityKeccak256,
-  toUtf8String,
-} from "./ethers";
+export * from "./ethers";
```

Comment:
should have done it like this from the start

Reply:
actaully im going to revert this because it makes it harder to discover the functions in each file when goign to the index page